python/felis/tap.py

```diff
@@ -202,6 +197,8 @@ def visit_schema(self, schema_obj: _Mapping) -> None:
             session.add_all(keys)
             session.add_all(key_columns)
 
+            logger.debug(f"Committing TAP schema: {schema_obj.name}")
+            logger.debug(f"TAP tables: {len(self.tables)}")
```

Comment:
It's better to avoid string formatting when using `logging`, saves few CPU cycles if `debug` is not enabled:

            logger.debug("Committing TAP schema: %s", schema_obj.name)
            logger.debug("TAP tables: %s", len(self.tables))